sources/packages/backend/apps/test-db-seeding/src/test-db-seeding.module.ts

```diff
@@ -0,0 +1,52 @@
+import { Module } from "@nestjs/common";
+import {
+  DatabaseModule,
+  DesignationAgreement,
+  Institution,
+  InstitutionLocation,
+  InstitutionUser,
+  InstitutionUserAuth,
+  InstitutionUserTypeAndRole,
+  User,
+} from "@sims/sims-db";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { DiscoveryModule, MetadataScanner } from "@nestjs/core";
+import { SeedExecutor } from "./seed-executors/seed-executor";
+import { DiscoveryService } from "@golevelup/nestjs-discovery";
+import { CleanDb } from "./clean-db/clean-db";
+import {
+  DesignationAgreementService,
+  InstitutionUserService,
+} from "./db-seeding/02-institution";
+import {
+  InstitutionHelperService,
+  UserTypeRoleHelperService,
+} from "./test-seed-helper-services";
+
+@Module({
+  imports: [
+    DatabaseModule,
+    TypeOrmModule.forFeature([
```

Comment:
Let's have this array of entities centralized and used by the `forRoot` and `forFeature`. Apparently, Nestjs needs it declared in the two places to allow the injection as used in this app.